apps/test/unit/code-studio/components/progress/UnitOverviewHeaderTest.js

```diff
@@ -1,7 +1,7 @@
 import {shallow} from 'enzyme'; // eslint-disable-line no-restricted-imports
 import React from 'react';
 
-import {UnconnectedUnitOverviewHeader as UnitOverviewHeader} from '@cdo/apps/code-studio/components/progress/UnitOverviewHeader';
+import {UnconnectedUnitOverviewHeader} from '@cdo/apps/code-studio/components/progress/UnitOverviewHeader';
```

Comment:
I really dislike this pattern of `UnconnectedUnitOverviewHeader as UnitOverviewHeader`. It obscures what is imported and requires the developer to look at the imports to understand that it is not using the connected version of the class.

Reply:
I agree